src/app/app.component.html

```diff
@@ -4,4 +4,8 @@
     label="Impersonate Chuck Norris"
     name="name"
     [(value)]="value"></app-input>
+  <app-custom-select
+    [placeholder]="'SelectCategory' | translate"
+    [options]="['dog', 'cat', 'bear']"
```

Comment:
keep array in ts file, 

`public options: string[] = ['dog', 'cat', 'bear']`

and in the template `[options]="options"` for example